pkg/server/server.go

```diff
@@ -7,9 +7,10 @@ import (
 	"time"
 
 	cache "github.com/chenyahui/gin-cache"
+	"github.com/chenyahui/gin-cache/persist"
 	sloggin "github.com/samber/slog-gin"
 
-	"github.com/chenyahui/gin-cache/persist"
+	// "github.com/chenyahui/gin-cache/persist"
```

Comment:
nit: This can be removed

Reply:
good catch

Reply:
Done! [`3d58b00` (#16)](https://github.com/seqeralabs/staticreg/pull/16/commits/3d58b001d97d686eeee30f50ff10119d9d80732f)